faces/4.1/_index.md

```diff
@@ -33,21 +32,19 @@ The removes references to the SecurityManager, further aligns with CDI where pos
 
 * [Jakarta Faces 4.1 Release Record](https://projects.eclipse.org/projects/ee4j.faces/releases/4.1)
```

Comment:
Most other specs also include a link to the Platform release plan:
```suggestion
* [Jakarta Faces 4.1 Release Record](https://projects.eclipse.org/projects/ee4j.faces/releases/4.1)
  * [Jakarta EE 11 Platform Release Plan](https://jakartaee.github.io/platform/jakartaee11/JakartaEE11ReleasePlan)
```

Reply:
Looks good, thanks Andy! :)